### ISSUE TITLE
Text input hint

### ISSUE THREAD

**Issue by mfikes** [replete-repl/replete-android#21]
Currently it is "Type in here".

I think an improvement could either be "Enter form here" or, alternatively, omitting the hint.

I'm thinking that iOS could have the same hint, but perhaps what occurs on iOS is that it is visually more clear that there is a text entry field given the way it looks. Comparing with the Messages app, Apple puts a hint of "iMessage" in that box. The equivalent for us would be something shorter, as in "Form", but that feels too short.

I wonder if the text entry form on Android could have a little chrome around it like the iOS one has. (And to be honest, I think the iOS text entry form could have its text entry form chrome "reduced" a bit, as the box feels a bit vertically narrow there.)

Anyway, in short this ticket identifies a difference, without really suggesting exactly what we should do, leaving it open for discussion.

**Comment by mfikes** [comment]
Yeah, I alluded to the possibility of omitting the hint in #14. The thing that I wonder is whether the resulting UI looks a little odd. 

I imagine a keyboard with a line above it and a cursor. Maybe this line is at the core of what I'm thinking. I'm perhaps expecting a box instead, but that is admittedly informed from the use of the Messages app. (I don't know if a line is "normal" on Android, and if a line alone sufficiently signals that a text input area exists.)

**Comment by roman01la** [comment]
I think this input field is a default Material Design field on Android, at least now that I am mostly Android user I see it frequently in other apps.

**Comment by roman01la** [comment]
I've updated hint text in de9e9b22
